src/WireMock.Net/Matchers/CSharpCodeMatcher.cs

```diff
@@ -151,7 +151,7 @@ private bool IsMatch(dynamic input, string pattern)
                 throw new WireMockException("CSharpCodeMatcher: Problem calling method 'IsMatch' in WireMock.CodeHelper", ex);
             }
 
-#elif (NETSTANDARD2_0 || NETSTANDARD2_1)
+#elif (NETSTANDARD2_0 || NETSTANDARD2_1 || USE_ASPNETCORE)
```

Comment:
I think it makes more sense to change to `#elif (NETSTANDARD2_0 || NETSTANDARD2_1 || USE_NETCOREAPP3_1)` ?

Reply:
Good idea! Would it be `USE_NETCOREAPP3_1` or `NETCOREAPP3_1`? I believe the second one is automatically created for us so it wouldn't require a change to the csproj.

Reply:
I'm gonna assume you meant `NETCOREAPP3_1` since your other comment had that. Let me know if you meant something different though!